skbase/validate/__init__.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: skbase developers, BSD-3-Clause License (see LICENSE file)
+"""Tools for validating and comparing BaseObjects and collections of BaseObjects."""
+import inspect
+from collections.abc import Iterable
+from typing import List
+
+__author__: List[str] = ["RNKuhns", "fkiraly"]
+__all__: List[str] = []
+
+
+def _check_list_of_str_or_error(arg_to_check, arg_name):
```

Comment:
I thought it was best practice to not have any functionality in `__init__` (except related to the concern of generating exports)

Reply:
Good catch, I was moving things around quickly. I've fixed this.